src/libraries/System.Drawing.Common/ref/System.Drawing.Common.netcoreapp.cs

```diff
@@ -26,6 +26,13 @@ public sealed partial class Graphics
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public void GetContextInfo(out PointF offset, out Region? clip) { throw null; }
     }
+    public sealed partial class Font
+    {
+        public void ToLogFont<T>(ref T logFont, Graphics graphics) where T: struct  { throw null; }
+        public static Font FromLogFont<T>(in T logFont) where T: struct { throw null; }
+        public static Font FromLogFont<T>(in T logFont, IntPtr hdc) where T: struct { throw null; }
+        public void ToLogFont<T>(ref T logFont) where T: struct { throw null; }
+    }
```

Comment:
Shouldn't this be conditionalised with `#if NET7_0_OR_GREATER`?


```
/__w/1/s/.packages/microsoft.dotnet.apicompat/7.0.0-beta.22255.2/build/Microsoft.DotNet.ApiCompat.targets(52,5): error : Compat issues with assembly System.Drawing.Common: [/__w/1/s/src/libraries/System.Drawing.Common/src/System.Drawing.Common.csproj]
##[error].packages/microsoft.dotnet.apicompat/7.0.0-beta.22255.2/build/Microsoft.DotNet.ApiCompat.targets(52,5): error : (NETCORE_ENGINEERING_TELEMETRY=Build) Compat issues with assembly System.Drawing.Common:
/__w/1/s/.packages/microsoft.dotnet.apicompat/7.0.0-beta.22255.2/build/Microsoft.DotNet.ApiCompat.targets(52,5): error : MembersMustExist : Member 'public System.Drawing.Font System.Drawing.Font.FromLogFont<T>(T)' does not exist in the implementation but it does exist in the contract. [/__w/1/s/src/libraries/System.Drawing.Common/src/System.Drawing.Common.csproj]
##[error].packages/microsoft.dotnet.apicompat/7.0.0-beta.22255.2/build/Microsoft.DotNet.ApiCompat.targets(52,5): error : (NETCORE_ENGINEERING_TELEMETRY=Build) MembersMustExist : Member 'public System.Drawing.Font System.Drawing.Font.FromLogFont<T>(T)' does not exist in the implementation but it does exist in the contract.
```

Reply:
Is there a reason to only have it on net7.0 and not on the net6.0 asset in both the ref and impl?

Reply:
I simply think that adding new API to .Net 7 would be less of a problem then anything else. Nothing prevent this API to work in .NET 6,

Reply:
When I as a customer intentionally upgrade my System.Drawing.Common package reference from 6.0.0 to 7.0.0, I would expect that I get new features irrelevant of the underlying target framework that is being used. I think it makes sense to improve both the net6.0 and the net7.0 assets if there's no underlying reason to not do so.

Reply:
That's good point. I forget about that scenario, even if I use it all the time.